config/config-example.js

```diff
@@ -64,19 +64,23 @@ exports.loginserverpublickey = "-----BEGIN RSA PUBLIC KEY-----\n" +
 // report joins and leaves - shows messages like "<USERNAME> joined"
 //   Join and leave messages are small and consolidated, so there will never
 //   be more than one line of messages.
+//   If this setting is set to `true`, it will override the client-side
+//   /hidejoins configuration for users.
 //   This feature can lag larger servers - turn this off if your server is
 //   getting more than 80 or so users.
 exports.reportjoins = true;
 
+// report joins and leaves periodically - sends silent join and leave messages in batches
+//   This setting will only be effective if `reportjoins` is set to false, and users will
+//   only be able to see the messages if they have the /showjoins client-side setting enabled.
+//   Set this to a positive amount of milliseconds if you want to enable this feature.
+exports.reportjoinsperiod = 0;
+
 // report battles - shows messages like "OU battle started" in the lobby
 //   This feature can lag larger servers - turn this off if your server is
 //   getting more than 160 or so users.
 exports.reportbattles = true;
 
-// report joins and leaves in battle - shows messages like "<USERNAME> joined" in battle
-// Turn this off on large tournament servers where battles get a lot of joins and leaves.
-exports.reportbattlejoins = true;
```

Comment:
I guess for now, we add that back. I'll remove it when I make that feature not suck.